backend/typescript/services/implementations/authService.ts

```diff
@@ -63,7 +63,7 @@ class AuthService implements IAuthService {
           firstName: googleUser.firstName,
           lastName: googleUser.lastName,
           email: googleUser.email,
-          roleId: RoleId.Staff,
+          role: "Staff",
```

Comment:
same here

Reply:
actually this doesn't matter we're gonna deprecate google sign in later anyway

Reply:
lol already did it, good catch lmao i got lazy and just used strings